srcDoc/release_notes.rst

```diff
@@ -80,6 +80,13 @@ Added RTI Perftest and RTI Connext DDS information at beginning of the test (#54
 Starting with this release, RTI Perftest will print at the beginning of the test
 its version and the version of RTI Connext DDS used to compile against.
 
+Automatically regenerate `qos_string.h` file if `perftest_qos_profiles.xml` is modified (#63)
```

Comment:
Add to the compilation documentation this new parameter.

Reply:
Done. @javmorcas  Can you review latest changes?